codes/block_definitions/mutate/block_mutate.py

```diff
@@ -84,6 +84,28 @@ def mutate(self, mutant_material: BlockMaterial, block_def): #: BlockDefinition)
             mutate_methods.mutate_single_ftn(mutant_material, block_def)
 
 
+class BlockMutate_SimGAN(BlockMutate_Abstract):
```

Comment:
I created an extra mutate block because I didn't want to automatically exclude the current operator in `mutate_methods.mutate_single_ftn`. This is just because I only have one operator for refiners at the moment (conv1d layer), and thus was hitting an error when mutating.

Reply:
Ah very true!